main-app/app/scripts/login/login-controller.js

```diff
@@ -0,0 +1,21 @@
+(function () {
+    'use strict';
+    angular.module('Tombola.BingoClient.Login')
+        .controller('LoginController', ['$scope', 'BingoApiService', '$state', 'UserData',
+            function($scope, bingoApiService, $state, userData){
+            var proxy = bingoApiService;
+
+            $scope.password = '';
+            $scope.username = '';
+
+            $scope.login = function(){
```

Comment:
This should be separated out from the controller into a service - defining this here is potentially the beginning of a slide into the "fat controller" scenario.